streams/src/test/java/org/apache/kafka/streams/state/internals/RocksDBKeyValueStoreTest.java

```diff
@@ -92,6 +92,7 @@ public void shouldUseCustomRocksDbConfigSetter() throws Exception {
         driver.setConfig(StreamsConfig.ROCKSDB_CONFIG_SETTER_CLASS_CONFIG, TheRocksDbConfigSetter.class);
         createKeyValueStore(driver.context(), Integer.class, String.class, false);
         assertTrue(TheRocksDbConfigSetter.called);
+        ((MockProcessorContext) driver.context()).close();
```

Comment:
This won't get called if the test fails. I think it would be better to convert `driver` into a field and call `close` in an `@After`